helm-charts/splunk-otel-collector/values.yaml

```
@@ -833,6 +833,14 @@ image:
     # The policy that specifies when the user wants the opentelemetry collector images to be pulled
     pullPolicy: IfNotPresent
 
+  universalBaseImage:
```

It doesn't have to UBI, can be any other linux image. So let's call it same as the container name for consistency with options in this section.
```suggestion
  # Image to be used by init container that patches log directories on the host, so the collector can read from them as a non-root user.
  # Effective only if `agent.securityContext.runAsUser` and `agent.securityContext.runAsGroup` are set to non-zero values.
  chown:
```

---

updated